CLAUDE.md

```
@@ -95,6 +98,265 @@ Three export paths:
+**With persistence**:
+```ts
+export const useQueriesStore = create<QueriesStore>()(
+  persist(
+    (set, get) => ({ ... }),
+    { name: "dbstudio-queries" }, // localStorage key
+  ),
+);
+```
+
+Rules:
+- Updates always produce new objects (immutable pattern via `set`)
```

markdownlint-cli2 (0.22.0)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)